README.md

```diff
@@ -539,14 +539,15 @@ container.loadModules([
   formatName: 'camelCase',
   // Apply resolver options to all modules.
   resolverOptions: {
-    // We can give these auto-loaded modules
-    // the deal of a lifetime! (see what I did there?)
-    // By default it's `TRANSIENT`.
-    lifetime: Lifetime.SINGLETON,
-    // We can tell Awilix what to register everything as,
-    // instead of guessing. If omitted, will inspect the
-    // module to determine what to register as.
-    register: awilix.asClass
+      // We can give these auto-loaded modules
```

Comment:
Noticed the indentation is off now; let's change line 552 instead from `)` to `})`

Reply:
Okay. But it will be line 551, as I see now.